tests/clients/test_mysqlclient.py

```diff
@@ -75,24 +71,22 @@ def test_vanilla_query(self):
         self.assertEqual(0, len(spans))
 
     def test_basic_query(self):
-        result = None
         with tracer.start_active_span('test'):
             result = self.cursor.execute("""SELECT * from users""")
             self.cursor.fetchone()
```

Comment:
If we just assert on the number of rows returned (i.e. result), we can remove the `cursor.fetch*` statements right?

Reply:
Good catch! This comment has revealed an even bigger issue.
Namely that the number of the rows in the database are completely out of control.
For example if you keep running the tests `test_mysqlclient.py`, then the number of rows keep increasing, because there is a TC which keeps inserting, and nothing deletes...
Also because the order in which the TCs are running are undefined, as they should not impact each other (but they do), currently the resul in ` result = self.cursor.execute("""SELECT * from users""")` is also undeterministic.
That is also the reason why previously there was no definite value `test_basic_query` like `result >= 0`, because it was impossible to tell with certainty. Now all that is fixed up in a new commit. Please check

Reply:
@Ferenc- 
The changes look perfect! Can we make similar changes to the other 2 files (test_pymysql.py, test_psycopg2.py) just to be consistent?

Reply:
Right. Now I have fixed those too.